From 521f40679b492c3b01d09fba62de2263e39c3e62 Mon Sep 17 00:00:00 2001
From: Randall Hand <rhand@magicleap.com>
Date: Mon, 4 Aug 2014 11:37:37 -0500
Subject: [PATCH] Removed final sprintf.

---
 ytnef/src/ytnefprint/main.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ytnef/src/ytnefprint/main.c b/ytnef/src/ytnefprint/main.c
index 2afcf62..7e086da 100644
--- a/ytnef/src/ytnefprint/main.c
+++ b/ytnef/src/ytnefprint/main.c
@@ -71,7 +71,7 @@ void PrintTNEF(TNEFStruct TNEF) {
     int j, object;
     int count;
     FILE *fptr;
-    char ifilename[256];
+    char ifilename[1024];
     char *charptr, *charptr2;
     DDWORD ddword_tmp;
     int SaveFile;
@@ -225,7 +225,7 @@ void PrintTNEF(TNEFStruct TNEF) {
                     object = 0;
                 }
             }
-            sprintf(ifilename, "%s", filename->data);
+            snprintf(ifilename, 1024, "%s", filename->data);
             for(i=0; i<strlen(ifilename); i++) 
                 if (ifilename[i] == ' ') 
                     ifilename[i] = '_';
